pkg/apis/core/v1beta1/validation/validation.go

```diff
@@ -268,6 +268,18 @@ func validateGreaterThan0(path *field.Path, value int64) field.ErrorList {
 }
 
 func ValidateKubeFedCluster(object *v1beta1.KubeFedCluster) field.ErrorList {
+	spec := object.Spec
+	specPath := field.NewPath("spec")
 	allErrs := field.ErrorList{}
+
+	disabledTLSValidationsPath := specPath.Child("disabledTLSValidations")
+	for i, value := range spec.DisabledTLSValidations {
+		if i > 0 && value == v1beta1.All {
```

Comment:
Per the comment in the API, this only verifies that `v1beta1.All` is first in the list, but not the only option in the list. Can we also validate that it's the first option and the rest of the list is empty?

Reply:
Updated the check to return err if v1beta1.TSLAll is specified in a list with more than one object.